docs/contributing/index.md

```diff
@@ -1,99 +1,88 @@
 
-```
-./scripts/docs-live.sh
-```
+=== "Contributing to Documentation"
 
-If you encounter any issues while setting up the documentation using the provided steps, please refer to our [tutorial]( https://www.youtube.com/watch?v=mNjlMZ4F3So) for additional guidance.
-
-##### Caveats and considerations
-
-Our mkdocs setup has a known limitation: the hot reloader won't auto-reload when changing docstrings. To see updates, manually rebuild and re-serve docs. We're working to improve the developer experience and appreciate your patience.
+    - Create a branch for your documentation changes in your fork
+    - Update the documentation [following our style guide](development.md#documentation)
+    - Preview the documentation [by rendering it locally](development.md#documentation). If you're not comfortable with this step, we'd rather you skip it and open a PR anyway! Our GitHub automations will generate a documentation preview. Please mark your PR as a draft until you've checked the preview and it looks OK. Don't hesitate to reach out for help!
+    - Describe your changes in the `CHANGELOG.md`
 
+Once you've completed these steps, you are ready to submit your pull request.
 
 ## Submitting your contribution
 
-- Run all unit tests successfully in your branch
-- Lint and format your code.  See below.
-- Update the documentation and CHANGELOG.md
-- Submit the fix to the problem as a pull request
-- Include an explanation of what you did and why in the pull request
-
-### Please format and lint as you go
-
-
-#### Usage of Pre-Commit
-
-To maintain code quality, we use pre-commit for automated checks before committing changes. Install and configure pre-commit to ensure high-quality contributions.
-
-To set up pre-commit, follow these steps:
-
-- `pip install pre-commit` ([official installation docs](https://pre-commit.com/#install))
-- `pre-commit install` to enable it to run automatically
-- `pre-commit run -a`  to run it manually
-
-
-We have included type stubs for the untyped `python-cmr` library, which we intend to eventually upstream.  Since `python-cmr` exposes the `cmr` package, the stubs appear under `stubs/cmr`.
+When you're ready to submit your pull request, first open a draft pull request and confirm that:
 
-## Pull Request process
+- You've included an explanation of what you did and why in the pull request
+- All unit tests run successfully in your branch
+- Your code is linted and formatted correctly
+- The documentation and `CHANGELOG.md` has been updated appropriately
 
-Fork the repository using the "Fork" button on the [repository
-homepage](https://github.com/nsidc/earthaccess), create a branch with your changes in the fork, then open
-a draft pull request from your fork. Starting a pull request provides additional instructions and requirements, and
-there is no harm in starting a draft pull request while still developing.
+Then you can mark the pull request as ready for review! :tada:
```

Comment:
```suggestion
Then you can mark the pull request as ready for review! 🎉
```
MkDocs apparently doesn't do `:tada:` -> 🎉 (there's probably an extension for that)